### ISSUE TITLE
Changing windows (in Tab Organizer) causes the windows to unmaximize.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#133]
Originally reported on Google Code with ID 134

```
Ryan:

I'm using latest beta (which is close to being the stable version) as well as Vista.
By the way when i say i switch tabs i mean i switch tabs within tab organizer going
from browsing in one window then selecting a tab in another window within tab organizer
and clicking on it (not within chrome itself). So every time i change windows using
tab organizer the window shrinks. This also happens with latest canary build also.
I don't recall this happening with previous version of tab organizer. Try following
these steps.  

1) Open chrome and tab organizer 
2) Open another window by click the wrench menu
3)Make sure both windows are expanded by clicking on the square next to the x button
in the right corner
4) Try switching between windows using tab organizer and see if it shrinks the window
```

Reported by `pcxunlimited` on 2011-02-06 02:10:22

**Comment by Pauan** [comment]
```
Thus far, it's only been reported for Windows... and it's not happening for me on GNU/Linux.

Also, see this video:
http://www.youtube.com/watch?v=usVvt7Jxky4
```

Reported by `pcxunlimited` on 2011-02-06 03:20:21

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2011-02-06 03:22:43
- **Labels removed**: OS-All

**Comment by Pauan** [comment]
```
This is probably issue chromium:65371.
```

Reported by `temp01irc` on 2011-02-09 01:40:22

**Comment by Pauan** [comment]
```
Yeah, that's it. Thanks!
```

Reported by `pcxunlimited` on 2011-02-09 01:43:52
- **Status changed**: `ExternalDependency`
- **Blocked on**: #65371

**Comment by Pauan** [comment]
```
Thanks for spotting the bug Temp01
```

Reported by `ryangalwey` on 2011-02-09 19:26:51

**Comment by Pauan** [comment]
```
Issue 137 has been merged into this issue.
```

Reported by `pcxunlimited` on 2011-02-20 05:59:47

**Comment by Pauan** [comment]
```
This was marked "Fixed" on the Chrome side a while back... is this still a problem in
the stable, beta, or dev channels?
```

Reported by `pcxunlimited` on 2011-04-27 08:01:01

**Comment by Pauan** [comment]
```
This has been marked as "Fixed" in Chrome for a long time, so I'm going to assume it's
fixed. If it's still a problem, please post in here or file a new issue, thanks.
```

Reported by `pcxunlimited` on 2012-05-01 01:45:51
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:48
- **Status changed**: `Fixed`
